### PR TITLE
Clarify which version of PowerShell Core supports which platform.

### PR DESCRIPTION
**Pull request by TravisEz13**
Clarify which version of PowerShell Core supports which platform.
Also, add details about EOL policy and when platforms where EOL'ed

The language about EOL  was taken from here: https://github.com/dotnet/core/blob/master/os-lifecycle-policy.md

Version(s) of document impacted
------------------------------
- [x] Impacts 6.1 document
- [x] Impacts 6.0 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/PowerShell-Core-Support.md](https://github.com/TravisEz13/PowerShell-Docs/blob/Add_support_matrix/reference/docs-conceptual/PowerShell-Core-Support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-3.0&branch=pr-en-us-2677) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-4.0&branch=pr-en-us-2677) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-5.0&branch=pr-en-us-2677) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-5.1&branch=pr-en-us-2677) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-6&branch=pr-en-us-2677) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C9%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201807092102141621-2677%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=VIinQdjnB4dgd7eZ9FIQ2SrryHB9u%2BUbfMZcfFNeCi8%3D&st=2018-07-09T21%3A09%3A13Z&se=2018-08-09T21%3A14%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by TravisEz13** [comment]
@DCtheGeek where are the rules for what branch to target.  This is actually more about accurately showing what we support for 6.0 as the previous document had added platforms which are only supported in 6.1.

**Comment by DCtheGeek** [comment]
@TravisEz13 I've reviewed and it can go in either, I just wanted to check.  Putting in staging means it'll go to live at next big publish (once or twice a week).  Putting it into release-61 means it isn't ready to be the official article yet, but when 6.1 is GA, then we'll merge that branch into live, thus publishing anything that's just waiting on 6.1.  Just let me know which you'd like and I'll approve.

File: reference/docs-conceptual/PowerShell-Core-Support.md [hunk 1]
```diff
@@ -33,26 +33,46 @@ Eventually, we expect PowerShell Core will adopt the "long-term servicing" appro
...
+| OpenSUSE 42.2                                     | Supported | Supported |
+| Fedora 27                                         | Supported | Supported |
+| Fedora 28                                         |           | Supported |
+| macOS 10.12+                                      | Supported | Supported |
 
 Our community has also contributed packages for the following platforms,
-but they are not officially suppported:
+but they are not officially supported:
 
 * Arch Linux
```
Comment (SteveL-MSFT) [inline reference/docs-conceptual/PowerShell-Core-Support.md]: Would it be easier to read if you put these in the table and have it say `Community`?
Should we include `Experimental` for Raspbian on 6.0 and `Supported` for 6.1?
We'll have to update this table again once Alpine docker image is out or just do it now.

Comment (TravisEz13) [reply reference/docs-conceptual/PowerShell-Core-Support.md]: Raspbian isn't supported by DotNet, so I say community:  https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1-supported-os.md#linux

I think we should update Alpine when the work is done.

I'll update the table as suggested.

**Comment by TravisEz13** [comment]
@DCtheGeek  I don't think there is a reason to hold this for the 6.0 release.  I think it's easier just to release the full thing to explain why Fedora 28 was changed.

**Comment by DCtheGeek** [comment]
@TravisEz13 Works for me.  Once Steve & Joey approve and the build passes, I'll push it forward.  Thanks!

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/PowerShell-Core-Support.md](https://github.com/TravisEz13/PowerShell-Docs/blob/Add_support_matrix/reference/docs-conceptual/PowerShell-Core-Support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-3.0&branch=pr-en-us-2677) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-4.0&branch=pr-en-us-2677) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-5.0&branch=pr-en-us-2677) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-5.1&branch=pr-en-us-2677) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-6&branch=pr-en-us-2677) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C10%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201807101747021556-2677%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Q8lQkp%2FaHTSs%2F27JppDUBk3%2Fo2QU5s8G%2BcrRRbU%2B6O4%3D&st=2018-07-10T17%3A53%3A12Z&se=2018-08-10T17%3A58%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/PowerShell-Core-Support.md](https://github.com/TravisEz13/PowerShell-Docs/blob/Add_support_matrix/reference/docs-conceptual/PowerShell-Core-Support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-3.0&branch=pr-en-us-2677) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-4.0&branch=pr-en-us-2677) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-5.0&branch=pr-en-us-2677) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-5.1&branch=pr-en-us-2677) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-6&branch=pr-en-us-2677) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C10%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201807101757532721-2677%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=4h3NuLItLiwvWZkCCuJzicj%2FHMl47y%2FCDDeYCR5H9AA%3D&st=2018-07-10T18%3A04%3A51Z&se=2018-08-10T18%3A09%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/docs-conceptual/PowerShell-Core-Support.md [hunk 2]
```diff
@@ -33,25 +33,48 @@ Eventually, we expect PowerShell Core will adopt the "long-term servicing" appro
...
-* Arch Linux
-* Kali Linux
-* AppImage (works on multiple Linux platforms)
+Please see the following table to see what platform the version of PowerShell Core you are using is officially supported.
+
+Our community has also contributed packages for some platforms,
+but they are not officially supported.
+These packages are marked as `Community` in the table.
+
+Platforms listed as `Experimental` are not officially supported, but is available for experimentation of feedback.
```
Comment (joeyaiello) [inline reference/docs-conceptual/PowerShell-Core-Support.md]: "...but are available for experimentation and feedback."

Comment (TravisEz13) [reply reference/docs-conceptual/PowerShell-Core-Support.md]: Thanks

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/PowerShell-Core-Support.md](https://github.com/TravisEz13/PowerShell-Docs/blob/Add_support_matrix/reference/docs-conceptual/PowerShell-Core-Support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-3.0&branch=pr-en-us-2677) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-4.0&branch=pr-en-us-2677) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-5.0&branch=pr-en-us-2677) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-5.1&branch=pr-en-us-2677) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-core-support?view=powershell-6&branch=pr-en-us-2677) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C11%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201807112205047170-2677%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=sx1IQyiNM9TXWvd7dVbbOsEG%2F3%2FJzBXKPk5GdOGkg4g%3D&st=2018-07-11T22%3A11%3A07Z&se=2018-08-11T22%3A16%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**